tests/unit/test_utils.py

```
@@ -0,0 +1,492 @@
+from datetime import datetime
+from unittest.mock import patch
+
+import click
+import pytest
+
+from toady.exceptions import ValidationError
+from toady.utils import MAX_PR_NUMBER, emit_error, parse_datetime
+
+
+@pytest.mark.unit
+class TestConstants:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)